charts/backup-pvc-cronjob/values.yaml

```diff
@@ -14,13 +14,12 @@ fullnameOverride: ""
 
 podAnnotations: {}
 
-podSecurityContext: {}
+podSecurityContext:
   # fsGroup: 2000
-
-securityContext: 
   runAsNonRoot: true
-  readOnlyRootFilesystem: true
-  # runAsUser: 8888
+  runAsUser: 1000
+
+securityContext: {}
 
 resources:
   limits:
```

Comment:
I know it's out of scope, but these CPU limits seem quite low since I see high CPU throttling for the job that's running right now in blackbalsam. Tar and gzip are quite CPU-hungry.